modules/bitgo/test/v2/unit/tss/ecdsa.ts

```diff
@@ -236,7 +207,8 @@ describe('test tss helper functions', function () {
           recipientPrivateArmor: userGpgKeypair.privateKey,
           senderPublicArmor: backupGpgKeypair.publicKey,
         }],
-      }).should.be.rejectedWith('Error decrypting message: Session key decryption failed.');
+        'nottherightkeychain',
+      ).should.be.rejectedWith('Error decrypting message: Session key decryption failed.');
```

Comment:
Not sure what is session key actually, may be the error should be something like decryption failed while combining, invalid private or public keys provided?